validation-test/stdlib/Unicode.swift.gyb

```diff
@@ -2380,7 +2380,7 @@ StringTests.test("StreamableConformance") {
   forStringsWithUnpairedSurrogates {
     (test: UTF16Test, subject: String) -> Void in
     let expected = test.scalarsHead + test.scalarsRepairedTail
-    let printedSubject = String(subject)
+    let printedSubject = subject
```

Comment:
Sorry, the point of this test was to test the Streamable conformance of String.  Now this test does nothing...